# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1475171623 14400
#      Thu Sep 29 13:53:43 2016 -0400
# Node ID 05ab1526377a644b92450028d7e2231f63b08c95
# Parent  b749559245a11e00321ea811b4aa365043913b7b
# Parent  acb3e59119d987592ae28258b52ba89cdfb6d990
maint: Merge stable to default.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -60,17 +60,17 @@ AC_SUBST(OCTAVE_API_VERSION)
 dnl FIXME: We should auto-insert the Mercurial changeset ID into the
 dnl        AC_REVISION field whenever configure.ac is modified.
 dnl AC_REVISION($Revision: 1.603 $)
 AC_CONFIG_SRCDIR([libinterp/octave.cc])
 AC_CONFIG_HEADERS([config.h:config.in.h])
 AC_CONFIG_AUX_DIR([build-aux])
 AC_CONFIG_MACRO_DIR([m4])
 
-AM_INIT_AUTOMAKE([1.11 foreign -Wno-portability -Wno-override tar-ustar subdir-objects])
+AM_INIT_AUTOMAKE([1.11 foreign -Wno-portability -Wno-override tar-ustar subdir-objects dist-lzip])
 
 ## Add the option to enable silent rules and make silent rules the
 ## default behavior.  Available since Automake 1.11 and included by
 ## default starting with Automake 1.13.
 AM_SILENT_RULES([yes])
 
 OCTAVE_CANONICAL_HOST
 
